web/locales/en.json

```
@@ -109,7 +109,9 @@
         "faq": "Frequently Asked Questions"
     },
     "wind": "wind",
+    "windDataError": "Wind data is unavailable",
     "solar": "solar",
+    "solarDataError": "Solar data is unavailable",
```

Maybe emphasise that this is only temporarily unavailable:

```suggestion
    "windDataError": "Wind data is currently unavailable",
    "solar": "solar",
    "solarDataError": "Solar data is currently unavailable",
```